frontend/src/components/login/InitialLoginStep.tsx

```diff
@@ -31,33 +33,60 @@ export default function InitialLoginStep({
 
     const handleLogin = async () => {
         try {
-          if (!email || !password) {
-            return;
-          }
-    
-          setIsLoading(true);
-          const isLoginSuccessful = await attemptLogin({
-            email,
-            password,
-          });
-          if (isLoginSuccessful && isLoginSuccessful.success) {
-            // case: login was successful
-    
-            if (isLoginSuccessful.mfaEnabled) {
-              // case: login requires MFA step
-              setStep(2);
-              setIsLoading(false);
-              return;
+            if (!email || !password) {
+                return;
             }
-    
-            // case: login does not require MFA step
-            router.push(`/dashboard/${localStorage.getItem('projectData.id')}`);
-          }
-    
+
+            setIsLoading(true);
+            const queryParams = new URLSearchParams(location.search)
+            if (queryParams) {
+                const callbackPort = queryParams.get("callback_port")
+
+                //attemptCliLogin
+                const isCliLoginSuccessful = await attemptCliLogin({
+                    email,
+                    password,
+                })
+
+                if (isCliLoginSuccessful && isCliLoginSuccessful.success) {
+                    // case: login was successful
+                    const cliUrl = `http://localhost:${callbackPort}`
+
+                    //send request to server endpoint
+                    const instance = axios.create()
+                    const cliResp = await instance.post(cliUrl,{...isCliLoginSuccessful.loginResponse,email,password})
```

Comment:
hmm, i wonder if this is safe to do to. Let me get back to this

Reply:
also since we are sending back login  2 response, that means that users will still have to retrun to CLI to enter their MFA code. I expected even MFA workflow to happen on the browser

Reply:
I wonder if we can somehow add a secure transport layer so that the data isn't just sent back over http to the local server

Reply:
Hm, I didn't consider the MFA workflow.
I don't believe that would be an issue(browser handling it) however what would be returned to the CLI?

Reply:
We return the same details back to the CLI  that would be retruned by the login2 call. You can see this being done here: https://github.com/Infisical/infisical/blob/main/cli/packages/cmd/login.go#L136. So what you can do is when it is MFA, have the user complete the MFA in the browser and once done, you can send back the details from the MFA verify call to the CLI

Reply:
Added details about what should be sent here https://github.com/Infisical/infisical/pull/648#issuecomment-1593815018

Reply:
In terms of security, we should be okay since we are communicating with the client itself